chain/core/api.py

```diff
@@ -179,6 +179,12 @@ def __init__(self, obj=None, queryset=None, data=None, request=None,
         self._request = request
         self._limit = limit or self.page_size
         self._offset = offset or 0
+        if self._queryset:
```

Comment:
Also - 

* refactor the `obj` argument to just take the ID as `obj_id`, and have the resource be responsible for creating the object out of the ID.
* remove the `queryset` argument to be a `list` boolean arg